tests/Controller/CRUDControllerTest.php

```diff
@@ -909,6 +910,50 @@ public function testDeleteAction(): void
         $this->assertSame('@SonataAdmin/CRUD/delete.html.twig', $this->template);
     }
 
+    public function testDeleteActionChildNoConnectedException(): void
```

Comment:
This test I removed it when cleaning up next major comments, but it just had to be converted from deprecations to exceptions. It showed as a conflicts and I rewrited it a little bit